### PR TITLE
[CLUSTER] `OS_DISTRIBUTION` is not set

### PR DESCRIPTION
**Pull request by tPl0ch**
Fixes https://github.com/kubernetes/kubernetes/issues/18746

### CODE REVIEW

**Comment by miry** [comment]
:+1:

**Comment by jlowdermilk** [comment]
Thanks for the pr. @k8s-bot ok to test

File: cluster/common.sh [hunk 1]
```diff
@@ -337,7 +337,7 @@ function find-release-tars() {
 
   # This tarball is only used by Ubuntu Trusty.
   KUBE_MANIFESTS_TAR=
-  if [[ "${OS_DISTRIBUTION}" == "trusty" ]]; then
+  if [[ "${KUBE_OS_DISTRIBUTION}" == "trusty" ]]; then
     KUBE_MANIFESTS_TAR="${KUBE_ROOT}/server/kuernetes-manifests.tar.gz"
```
Comment (jlowdermilk) [inline cluster/common.sh]: looks like a typo here as well: s/`kuernetes`/`kubernetes`

**Comment by tPl0ch** [comment]
@jlowdermilk Fixed also the typo and rebased against master.

**Comment by jlowdermilk** [comment]
lgtm, thanks @tPl0ch

**Pull request merged**